src/Version.php

```diff
@@ -32,6 +35,11 @@ class Version implements JsonSerializable
     /** @var Build */
     protected $build;
 
+    public static function setComparator(?ComparatorInterface $comparator) : void
```

Comment:
Similar as above: put these method above `protected function getComparator()`, to keep related functionalities grouped and closer to each other.